pkg/plugins/manager/loader/loader.go

```diff
@@ -263,6 +263,14 @@ func (l *Loader) createPluginBase(pluginJSON plugins.JSONData, class plugins.Cla
 		return nil, err
 	}
 
+	// Hardcode alias features
+	switch pluginJSON.ID {
```

Comment:
@wbrowne -- I removed the setting from the definition, and now set it manually here

Reply:
Why is this set here? I thought the `alias` property in plugin.json made sense.

Reply:
@aocenas -- I agree, but as a first step @wbrowne suggests that we only aim to support this one plugin